src/com/facebook/buck/android/RobolectricTestDescription.java

```diff
@@ -105,6 +105,7 @@ public Arg createUnpopulatedConstructorArg() {
         javacOptions,
         DependencyMode.TRANSITIVE,
         /* forceFinalResourceIds */ true,
+        Optional.<String>absent(),
```

Comment:
Comment.  I wonder if you blame the line below, if it would be from me.  How embarrassing.  I blame automated refactoring.